catalyst_voices/web/index.html

```diff
@@ -2,43 +2,42 @@
 <html>
 
 <head>
-  <!--
-    If you are serving your web app in a path other than the root, change the
-    href value below to reflect the base path you are serving from.
-
-    The path provided below has to start and end with a slash "/" in order for
-    it to work correctly.
-
-    For more details:
-    * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base
-
-    This is a placeholder for base href that will be replaced by the value of
-    the `--base-href` argument provided to `flutter build`.
-  -->
-  <base href="$FLUTTER_BASE_HREF">
-
-  <meta charset="UTF-8">
-  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
-  <meta name="description" content="Catalyst Voices">
-
-  <!-- iOS meta tags & icons -->
-  <meta name="apple-mobile-web-app-capable" content="yes">
-  <meta name="apple-mobile-web-app-status-bar-style" content="black">
-  <meta name="apple-mobile-web-app-title" content="Catalyst Voices">
-  <link rel="apple-touch-icon" href="icons/Icon-192.png">
-
-  <!-- Favicon -->
-  <link rel="icon" type="image/png" href="favicon.png" />
-
-  <title>Catalyst Voices</title>
-  <link rel="manifest" href="manifest.json">
-  <script type="module" src="/assets/packages/catalyst_cardano_web/assets/js/catalyst_cardano.js"></script>
-  <script type="application/javascript" src="/assets/packages/flutter_inappwebview_web/assets/web/web_support.js"
-    defer></script>
+    <!--
+      If you are serving your web app in a path other than the root, change the
+      href value below to reflect the base path you are serving from.
+
+      The path provided below has to start and end with a slash "/" in order for
+      it to work correctly.
+
+      For more details:
+      * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base
+
+      This is a placeholder for base href that will be replaced by the value of
+      the `--base-href` argument provided to `flutter build`.
+    -->
+    <base href="$FLUTTER_BASE_HREF">
+
+    <meta charset="UTF-8">
+    <meta content="IE=Edge" http-equiv="X-UA-Compatible">
+    <meta name="description" content="Catalyst Voices">
+
+    <!-- iOS meta tags & icons -->
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+    <meta name="apple-mobile-web-app-title" content="Catalyst Voices">
+    <link rel="apple-touch-icon" href="icons/Icon-192.png">
+
+    <!-- Favicon -->
+    <link rel="icon" type="image/png" href="favicon.png"/>
+
+    <title>Catalyst Voices</title>
+    <link rel="manifest" href="manifest.json">
+    <script type="module"
+            src="/assets/packages/catalyst_cardano_web/assets/js/catalyst_cardano.js"></script>
 </head>
 
 <body>
-  <script src="flutter_bootstrap.js" async></script>
+<script src="flutter_bootstrap.js" async></script>
```

Comment:
This formatting seems strange, this line should be indented compared to <body>. Can you revert formatting changes or configure your formatter to achieve the same formatting as previously?

Reply:
Hmm, I have standard formatting enabled and formatting job on CI passes too... so I guess that's what we need?

Reply:
Formatting job on CI doesn't check html files. We need unified formatting, otherwise somebody with different IDE will just reformat it again

Reply:
Formatting rules should possibly be close to how flutter formats this file when the project is generated. Here you formatted it with the format that you happened to have in your IDE. Then, a third person yet with another formatting will reformat it again in some time. Every time it happens there's an increased chance of conflicts with other people's formatting and it obscures the PR, the lines you really changed and the lines you just formatted.

Reply:
I think you can just apply your changes without reformatting, then we can discuss which format we want to use.